### ISSUE TITLE
Add to readme - jargon buster

### ISSUE THREAD

**Issue by JillJarvis** [AI4VBH/AIDE#25]
@heyhaleema From the review of the training docs on 10th Jan. Add to readme a jargon buster
E.g. workflow, DICOM ECHO, study etc

**Comment by Lucrezia-Cester** [comment]
What do we mean here for jargon buster? Is this supposed to mimic a glossary ?

**Comment by JillJarvis** [comment]
> What do we mean here for jargon buster? Is this supposed to mimic a glossary ?

Yes, that's right Lu, a glossary of terms

**Comment by Lucrezia-Cester** [comment]
@JillJarvis @heyhaleema In my opinion, a glossary should be its own section at the beginning of the documentation, whilst the README should include a brief high-level overview of explanation of main concepts and how they are interconnected so that one can familiarise themselves before delving in. In my opinion, that is going to be the most beneficial for users as their first cold approach to learn the material as easily as possible.

**Comment by heyhaleema** [comment]
@Lucrezia-Cester Agreed, let's go with the approach of having a separate glossary, reference some of the concepts in the README, and include links to relevant repos/docs that go into greater detail.

**Comment by JillJarvis** [comment]
@Lucrezia-Cester I agree with you and Haleema :-)
